config/config.php

```diff
@@ -12,24 +12,15 @@
  * Obviously, if you use closures in your config you can't cache it.
  */
 
-$cachedConfigFile = 'data/cache/app_config.php';
+$configManager = new ConfigManager([
+    // Load config from all modules first
+    new PhpFileProvider('src/*/Config/{{,*.}global,{,*.}local}.php'),
```

Comment:
@xtreamwayz 

Is it correct that you want to place config files in the `/src/` dir of each module? And that it is possible to have `global.php` and `local.php` within the modules? 

Why?